borrowd/config/base.py

```diff
@@ -162,6 +162,11 @@
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
 
+#
+# Borrowd settings
+#
+BORROWD_GROUP_INVITE_EXPIRY = 60 * 60 * 24 * 7  # 1 week
```

Comment:
Love the clear math instead of a big integer that leaves the reader to figure it out. One nit here: could this be renamed to `BORROWD_GROUP_INVITE_EXPIRY_SECONDS`? That just makes it all the more clear when using the variable.